domain.go

```diff
@@ -26,6 +26,29 @@ type DomainInput struct {
 	Note        *string `json:"note,omitempty"`
 }
 
+func (d *Domain) GetTag(client *Client, tagId ID) (*Tag, error) {
+	if d == nil {
```

Comment:
are you able to make this happen?  My understanding is if you do
```go
var myDomain *Domain
myDomain.GetTag()
```
you won't even get into this function at all you'll get a `GetTag() function doesn't exist on "nil" object` error

Reply:
Oh, [it's possible](https://go.dev/play/p/w-fpHemL7Zt)